### ISSUE TITLE
"Query" option

### ISSUE THREAD

**Issue by hejops** [aeyoll/soulseek-cli#16]
I mostly use soulseek to check for obscure or relatively new releases, both of which tend to return no results. Negative results already produce exit code 255, while positive results prompt for user input, which is not so friendly for scripting.

Would it be possible to add an option to simply query the database and tell the user whether the search term returns any results?

A simple example would be:
```
soulseek query 'something that exists'
>>> 100 results found
>>> Top result: ...
>>> exit 0

soulseek query 'something that does not'
>>> 0 results found
>>> exit 1
```

**Comment by aeyoll** [comment]
Hey @hejops,
That's a good idea, I'll add this feature!

**Comment by aeyoll** [comment]
@hejops Feature has been added in version 0.1.0. Can you fetch the latest version and give it a try? Thanks a lot!

**Comment by hejops** [comment]
It works great, thank you.

I recently made a small improvement to print the number of search results. I'll make a PR once I fork the repo and figure it out. :)

**Comment by aeyoll** [comment]
Merged your PR #17, thanks a lot for your contributions 😃
